tensorflow_recommenders_addons/dynamic_embedding/python/ops/cuckoo_hashtable_ops.py

```diff
@@ -264,8 +281,43 @@ def insert(self, keys, values, name=None):
       values = ops.convert_to_tensor(values, self._value_dtype, name="values")
       with ops.colocate_with(self.resource_handle):
         # pylint: disable=protected-access
-        op = cuckoo_hashtable_ops.tfra_cuckoo_hash_table_insert(
-            self.resource_handle, keys, values)
+        op = cuckoo_ops.tfra_cuckoo_hash_table_insert(self.resource_handle,
+                                                      keys, values)
+    return op
+
+  def accum(self, keys, values_or_deltas, exists, name=None):
```

Comment:
Seems it only registers CPU implementation. Need to raise a `NotImplementedError` when using GPU.